# Use double for floating point (64 bit) in METIS by default
# OpenFOAM 4 uses doubles be default
diff -ur metis-5.1.0.orig/include/metis.h metis-5.1.0/include/metis.h
--- metis-5.1.0.orig/include/metis.h	2013-03-30 17:24:45.000000000 +0100
+++ metis-5.1.0/include/metis.h	2016-09-05 11:39:33.004384533 +0200
@@ -40,7 +40,7 @@
    32 : single precission floating point (float)
    64 : double precission floating point (double)
 --------------------------------------------------------------------------*/
-#define REALTYPEWIDTH 32
+#define REALTYPEWIDTH 64



